### ISSUE TITLE
[Enhancement] Make Pf2e loot actors "Undetected"

### ISSUE THREAD

**Issue by weepingminotaur** [Saibot393/perceptive#22]
Currently, you can't apply the Undetected condition to a loot actor, which means that you can't make loot actors discoverable via Perceptive. Any way around this? It's a very common use case to have a loot actor containing swag that the party only finds if they pass a Perception check of a certain DC.

I will stop logging github issues now before you kill me. :)

**Comment by Saibot393** [comment]
I think the easiest way to do this is by using the build in foundry invisibility (top right icon on the token HUD). Since the gm will probably always want to set loot tokens perception dc by hand i don't think there is much value in auto applying this when rolling a stealth check. There may however be some problematic behaviour with some senses, not quite sure yet.

**Comment by weepingminotaur** [comment]
Oh, I wasn't asking for auto-rolls! Totally get that it's counterproductive. But would it be possible to set a loot actor's active and passive Perception DCs by hand, just like you can do for secret doors? That way when searching a room during exploration, you'd find secret doors and hidden treasure.

**Comment by Saibot393** [comment]
I will put a filter in place that excludes loot actors from the Pf2e perception dc/stealth automations.

**Comment by Saibot393** [comment]
Yes, thats the current behaviour (and will probably stay that way). Though since the loot token is probably made invisible through the foundry effect/setting, this effect can currently not be removed by perceptive, should the token be spotted by a player (even if Make spotted tokens visible is active)

**Comment by Saibot393** [comment]
It seems like loot actors in Foundry can not gain any effects , not even through console commands/code. I can therefore probably not make it work with the pf2e effects.
There is also a chance that if could somehow skip this limitation some strange stuff could happen, like player bing able to loot the undetected effect from loot actors.

**Comment by weepingminotaur** [comment]
Yeah, exactly, loot actors can't gain conditions. Any chance Perceptive could toggle the Foundry system visibility for a loot actor if it's successfully spotted?

**Comment by Saibot393** [comment]
Just tested it, shouldn't be a problem

**Comment by Saibot393** [comment]
This feature is now implemented with the new v2.0.0
